docs/source/building_applications/responses_vs_agents.md

```diff
@@ -0,0 +1,88 @@
+# Agents vs OpenAI Responses API
+
+Llama Stack provides two different APIs for building AI applications with tool calling capabilities: the **Agents API** and the **OpenAI Responses API**. While both enable AI systems that can use tools, they serve different use cases and have distinct characteristics.
+
+## Overview
+
+### Agents API
+The Agents API is a full-featured, stateful system designed for complex, multi-turn conversations with persistent sessions. It provides comprehensive agent management, detailed execution tracking, and rich metadata about each interaction.
+
+### OpenAI Responses API
+The OpenAI Responses API is a lightweight, stateless system that provides a simple interface for single-turn tool calling. It's designed to be compatible with OpenAI's API patterns while adding Llama Stack's tool calling capabilities.
+
+## Key Differences
+
+| Feature | Agents API | OpenAI Responses API |
+|---------|------------|---------------------|
+| **State Management** | Full session persistence with turns, steps, and metadata | Stateless - each request is independent |
+| **Complexity** | High - complete agent lifecycle management | Low - simple request/response pattern |
```

Comment:
Can we use something other than "complexity"? Even if Agent is more powerful, this shouldn't mean it's more complex.

Reply:
+1 to this. You're right, thanks!